autobahn/asyncio/wamp.py

```diff
@@ -172,3 +172,106 @@ def create():
             loop.run_until_complete(protocol._session.leave())
 
         loop.close()
+
+
+# TODO - unify with previous class
+class ApplicationRunnerRawSocket(object):
+    """
```

Comment:
A better approach would be to teach `ApplicationRunner` how to establish different transports ("better" as in until we get anywhere with the "new API" discussion;)